docs/en/05_Contributing/04_Release_Process.md

```diff
@@ -128,16 +128,11 @@ In the event of a confirmed vulnerability in our
 [supported modules](https://www.silverstripe.org/software/addons/silverstripe-commercially-supported-module-list/),
 we will take the following actions:
 
-*  Acknowledge to the reporter that we’ve received the report and that a fix is forthcoming. We’ll give a rough
-timeline and ask the reporter to keep the issue confidential until we announce it.
-*  Assign a unique identifier to the issue in the format `SS-<year>-<count>`, 
-   where `<count>` is a padded three digit number counting issues for the year. 
-   Example: `SS-2013-001` would be the first of the year `2013`.
-   Additionally, [CVE](http://cve.mitre.org) numbers are accepted.
-*  Halt all other development as long as is needed to develop a fix, including patches against the current and one
-previous major release (if applicable).
-* Pre-announce the upcoming security release to aprivate pre-announcement mailing list of important stakeholders (see below).
-*  We will inform you about resolution and [announce](https://forum.silverstripe.org/c/releases) a 
+* Acknowledge to the reporter that we’ve received the report and that a fix is forthcoming.
+  We’ll give a rough timeline and ask the reporter to keep the issue confidential until we announce it.
+* Assign a [CVE identifier](https://cve.mitre.org) to the issue.
+* For "high" and "critical" issues: Pre-announce the upcoming security release to aprivate pre-announcement mailing list of important stakeholders (see below).
```

Comment:
Perhaps say `For "high" and "critical" issues (CVSS > 7):` here. You can figure that out from the table below but it'd make the text more easily understood if you were familiar with CVSS scores.

Reply:
Sweet, fixed.